### PR TITLE
RANCHER-1183 Write a README (documentation) for folio-helm charts usage

### PR DESCRIPTION
**Pull request by epam-avramenko**
https://folio-org.atlassian.net/browse/RANCHER-1183

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,16 +1,271 @@
...
+            ├── _helpers.tpl
+            ├── ...............
+            └── _volume.tpl
+```
+
+## Usage
+### Chart usage
+
+The charts in this repository allow users to deploy different modules of the FOLIO project to a Kubernetes cluster using
+the Helm package manager. To do this you can follow the following steps:
```
Comment (OHaimanov) [inline README.md:74]: "follow these steps" or "use the following steps"

File: README.md [hunk 2]
```diff
@@ -1,16 +1,271 @@
...
+The `folio-common` helpers are included in the `templates/application.yaml` file.
+
+When creating new charts, use the existing pattern and make sure to cater for any unique characteristics
+that the new module might have. But also it give a pattern that needs to be followed.
+
+This approach ensures seamless deployments and maintenance of the FOLIO project modules.
+
+Please DO NOT forget to increase the Chart version if changes were made.
+
+After merging Pull Request to the master branch the job https://jenkins-aws.indexdata.com/job/Rancher/job/Index-helm-charts/
```
Comment (OHaimanov) [inline README.md:97]: Please change to https://jenkins-aws.indexdata.com/job/folioRancher/job/folioDevOpsTools/job/indexFolioHelmCharts/

File: README.md [hunk 3]
```diff
@@ -1,16 +1,271 @@
...
+- Kubernetes 1.22+
+- Helm 3.10+
+
+## Getting started
+
+To add the repository locally, execute the following command before installing the first chart.
+This command isn't necessary for subsequent chart installations.
+
+```console
+helm repo add folio https://repository.folio.org/repository/folio-helm-v2/
```
Comment (OHaimanov) [inline README.md:111]: Let's keep local repo name same as remote folio->folio-helm-v2

File: README.md [hunk 4]
```diff
@@ -1,16 +1,271 @@
...
+```
+
+## Install Chart
+
+Should the repository have been added a while ago, run the following command to update the repository cache
+with the latest changes before installing any chart.
+
+```console
+helm repo update folio
```
Comment (OHaimanov) [inline README.md:120]: Same folio->folio-helm-v2

File: README.md [hunk 5]
```diff
@@ -1,16 +1,271 @@
...
+```
+
+To install the chart invoke the following command.
+
+```console
+helm install my-install-name folio/CHART_NAME
```
Comment (OHaimanov) [inline README.md:126]: Same folio->folio-helm-v2

File: README.md [hunk 6]
```diff
@@ -1,16 +1,271 @@
...
+
+> **Note**: Please make sure to replace `CHART_NAME` with the name of the chart that needs to be installed.
+> The --version x.x.x attribute can be added at the end of the command to install a specific version of the chart.
+> Otherwise, the latest version will be installed.
+
+To get the exact name of a specific module chart, invoke the following command to see the full list of charts
+and their latest version in the folio repository:
+
+```console
+helm search repo folio
```
Comment (OHaimanov) [inline README.md:137]: Same folio->folio-helm-v2

**Review by OHaimanov — CHANGES_REQUESTED** [review]
Overall good, please fix comments

File: README.md (hunk 1 above)
Comment (epam-avramenko) [reply README.md]: Have done.

File: README.md (hunk 2 above)
Comment (epam-avramenko) [reply README.md]: Have done.

File: README.md (hunk 4 above)
Comment (epam-avramenko) [reply README.md]: Have done.

File: README.md (hunk 5 above)
Comment (epam-avramenko) [reply README.md]: Have done.

File: README.md (hunk 6 above)
Comment (epam-avramenko) [reply README.md]: Have done.

**Review by eldiiar-duishenaliev — APPROVED** [review]
All is ok.

**Review by OHaimanov — APPROVED** [review]

**Pull request merged**